Userland/Libraries/LibUnicode/CharacterTypes.h

```diff
@@ -68,4 +68,32 @@ bool code_point_has_grapheme_break_property(u32 code_point, GraphemeBreakPropert
 bool code_point_has_word_break_property(u32 code_point, WordBreakProperty property);
 bool code_point_has_sentence_break_property(u32 code_point, SentenceBreakProperty property);
 
+// https://www.unicode.org/reports/tr9/#Table_Bidirectional_Character_Types
+enum class BidirectionalCharacterType : u8 {
```

Comment:
This name is a bit unwieldy, but I don't know if we use any abbreviations in LibUnicode.

Reply:
Unwieldy names are fine :) you could call this "BidirectionalClass" to save some keys, as that's another canonical name for this data.